example/src/thunks/presentation.ts

```diff
@@ -101,13 +104,23 @@ export const remoteCrossDevicePresentationThunk = createAppAsyncThunk<
 
   const credentialCryptoContext = createCryptoContextFor(pid.keyTag);
 
-  const authResponse = await Credential.Presentation.sendAuthorizationResponse(
-    requestObject,
-    presentationDefinition,
-    jwks.keys,
-    [pid.credential, disclosuresRequestedClaimName, credentialCryptoContext],
-    { appFetch: appFetch }
-  );
+  let authResponse;
+
+  if (args.allowed === "ACCEPT") {
+    authResponse = await Credential.Presentation.sendAuthorizationResponse(
+      requestObject,
+      presentationDefinition,
+      jwks.keys,
+      [pid.credential, disclosuresRequestedClaimName, credentialCryptoContext],
+      { appFetch: appFetch }
+    );
+  } else {
+    authResponse = await Credential.Presentation.sendAuthorizationErrorResponse(
+      requestObject,
+      { error: "access_denied" },
+      { appFetch: appFetch }
+    );
+  }
```

Comment:
We avoid as much as possible the use of `let`.
```suggestion
const authResponse = args.allowed === "ACCEPT"
  ? await Credential.Presentation.sendAuthorizationResponse(
      requestObject,
      presentationDefinition,
      jwks.keys,
      [pid.credential, disclosuresRequestedClaimName, credentialCryptoContext],
      { appFetch: appFetch }
    )
  : await Credential.Presentation.sendAuthorizationErrorResponse(
      requestObject,
      { error: "access_denied" },
      { appFetch: appFetch }
    );
```

Reply:
A solution proposal has been pushed with commit 5307f67.